src/main/java/racing/domain/Cars.java

```diff
@@ -3,20 +3,26 @@
 import racing.domain.strategies.CarMoveStrategy;
 import racing.dto.GameResult.GameRoundResult;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class Cars {
     private final List<Car> cars;
 
-    public Cars(List<Car> cars) {
-        this.cars = cars;
+    public Cars(List<String> nameOfCars) {
+        cars = nameOfCars.stream()
+                .map(Car::new)
+                .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
```suggestion
                .collect(Collectors.toList());
```
이렇게도 작성할 수 있을 텐데 위와같이 작성하신 이유가 있을까요?